jupyter-notebooks/Subscriptions-API/Subscriptions_API_Quickstart.ipynb

```diff
@@ -65,164 +140,248 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "99d2453c-2e6b-450b-821e-5ed083346cb1",
+   "metadata": {},
+   "source": [
+    "#### Cloud Delivery Parameters:\n",
+    "\n",
+    " Your delivery or hosting parameters may be specified here. Below is a template for delivery to a Google Cloud Storage bucket. Note that you will need a serivce account with create, delete, and get persmissions and it is recommended to store your GCS credentials as an environment variable. See the [Google Cloud documentation](https://cloud.google.com/iam/docs/keys-create-delete#iam-service-account-keys-create-console) on how to create a service account key with the appropriate permissions. It is recommended to set the GCS credentials as an environment variable; more information is available [here](https://docs.planet.com/develop/apis/subscriptions/delivery/#google-cloud-storage). \n",
+    " <br> <br>\n",
+    " Other options may be found with `subscription_request.amazon_s3`, `subscription_request.oracle_cloud_storage`, etc...\n"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "36b3a567",
+   "id": "d91291a9-52b0-43df-bd74-ba7f52e3f6f8",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Note: if this shows an empty list \"[]\", this may mean that you don't have any currently running or pending subscriptions!\n",
-    "sub_list"
+    "# Google Cloud Storage placeholder\n",
+    "gcs_key = os.getenv('GCS_CREDENTIALS')\n",
+    "bucket_name = 'your_bucket_name'\n",
+    "# optional path_prefix\n",
+    "path_prefix = ''\n",
+    "bucket_delivery = subscription_request.google_cloud_storage(\n",
+    "    credentials=gcs_key, bucket=bucket_name, path_prefix=path_prefix)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "fffa6e0e",
+   "id": "023188bf-0fe6-4506-8e51-3084902cd63b",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "#### Alternative to cloud delivery: Sentinel Hub Hosting\n",
+    "\n",
+    "You may wish to host your subscription data with a Cloud hosting service like a Sentinel Hub Collection or Google Earth Engine. The `hosting` arg should be used *instead* of the `delivery` arg below in `create_subscription()`. \n",
+    "<br> <br>\n",
+    "For using Sentinel Hub collections, note that your Planet and Sentinel Hub accounts should be linked. Cloud hosting documentation for subscriptions can be found [here](https://docs.planet.com/develop/apis/subscriptions/delivery/#hosting).\n",
+    "<br> <br>\n",
+    "If you do not have a collection set up already, set `create_config` to True, and pass in `collection_id = None`. The collection will be created for you, with the name you use for the subscription. "
```

Comment:
Alternatively, you can just delete both those lines - the default is to create a new collection (also, create_config is not directly related to collection creation - it's for creating [configurations](https://docs.planet.com/platform/get-started/analyze-data/#configurations), which are different - I'd omit mentioning that property altogether, for either pattern)